sql/driver/conn.go

```diff
@@ -111,6 +108,12 @@ func (c *conn) send(args Request) (*rows, error) {
 	if index < 0 {
 		return r, nil
 	}
+	// Check for any application errors.
+	for _, result := range resp.Results {
+		if result.Error != nil {
+			return nil, result.Error
```

Comment:
we should investigate what the Go drivers for other databases do in this situation (the `psql` CLI client appears to handle each statement individually, both in error handling and result printing).

Could you add a TODO?